examples/directml/squeezenet/squeezenet_config.json

```diff
@@ -23,6 +23,12 @@
             }
         }
     },
+    "data_configs": [
+        {
+            "name": "perf_tuning_data_config",
```

Comment:
the type says dummydatacontainer but there is no configuration for any of the components. is this a valid data config?

Reply:
Yes, this would use `dummy_dataset` and `no_auto_batch_dataloader` because those are setup as default values in the definition of `DummyDataContainer`.

Reply:
doesn't the dummydatacontainer require `input_shapes` in load data component?

Reply:
Depends on the use case. When used in the context of perf tuning, this information comes from `input_model/io_config`.
Here in this case, the data config is used only for perf tuning.

Reply:
I updated the logic in `OliveEvaluator` to handle this scenario. In my opinion, providing a built-in definition for `EmptyContainer` is not a good idea. Such implementation will push users to do the minimum (i.e. implement the `DataContainer`) and break the required modularity. Such implementation will also prevent us from implementing any possible optimizations, in future, that would bank heavily on these components being modular.

Reply:
Removed.